src/program/lwaftr/doc/benchmarking.md

```diff
@@ -10,12 +10,20 @@ example assumes NICs `02:00.0` and `02:00.1` and wired to NICs `02:00.0` and
 Please, change the PCI addresses to match the settings in your current system; 
 See [running.md](running.md).
 
+Make a configuration with the PCI devices we're using:
+
+```
+$ sudo ./src/snabb lwaftr migrate-configuration -f pci-device \
+    -o "from[device=test]" -o "internal[device=0000:02:00.0]" \
+    -o "external[device=0000:02:00.1]" \
+    src/program/lwaftr/tests/data/icmp_on_fail.conf >> /tmp/icmp_on_fail.conf
+````
+
 In one server, start the lwAFTR:
 
 ```
 $ sudo numactl -m 0 taskset -c 1 ./src/snabb lwaftr run -v \
-    --conf program/lwaftr/tests/data/icmp_on_fail.conf \
-    --v4 0000:02:00.0 --v6 0000:02:00.1
+    --conf /tmp/icmp_on_fail.conf
```

Comment:
About this.  You mention that you're hesitant to keep supporting --v4 and --v6 given that we're looking to support multiple processes.  However given that we run the same softwire configuration all the time over different interfaces, and we have code that can migrate files in memory, it would certainly make sense to keep supporting --v4 / --v6 in single-interface mode; and in multiple-interface mode, you could specify `--v4=03:00.0,03:00.1` for example and that can make sense.  What do you think here?